idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/tests/test_CLC_gas_prop.py

```diff
@@ -66,13 +66,145 @@ def test_build_inlet_state_block(gas_prop):
     assert isinstance(gas_prop.fs.unit.cp_mol, Var)
     assert isinstance(gas_prop.fs.unit.cp_mass, Var)
     assert isinstance(gas_prop.fs.unit.visc_d, Var)
+    assert isinstance(gas_prop.fs.unit.enth_mol, Var)
+    assert isinstance(gas_prop.fs.unit.entr_mol, Var)
 
 
 @pytest.mark.unit
 def test_setInputs_state_block(gas_prop):
     assert degrees_of_freedom(gas_prop.fs.unit) == 0
 
 
+@pytest.fixture(scope="class")
+def gas_prop_unscaled(gas_prop):
```

Comment:
Rather than copy the model, I would suggest you just rebuild the model for each case. It does not cost that much to build a model, and copying the model runs the risk of test failures due to something going wrong during copying.

Reply:
Okay, I will update the fixtures. Would you recommend rebuilding the model rather than copying for the MB and BFB tests as well?